starboard/build/config/BUILD.gn

```diff
@@ -80,6 +80,7 @@ config("host") {
         }
       }
     } else if (is_win) {
+      configs = [ "//build/config/win:visual_studio_version_compat" ]
```

Comment:
I think it would be better add this here https://github.com/youtube/cobalt/blob/main/starboard/build/config/win/BUILD.gn#L22

I think this config should be as minimal as possible and probably pared down from what it is

Reply:
Ack